networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/peers/DefaultEth2Peer.java

```diff
@@ -453,7 +453,7 @@ public SafeFuture<Void> requestDataColumnSidecarsByRange(
 
               if (startSlot.isLessThan(firstSupportedSlot)) {
                 LOG.debug(
-                    "Requesting data column sidecars from slot {} instead of slot {} because the request is spanning the Deneb fork transition",
+                    "Requesting data column sidecars from slot {} instead of slot {} because the request is spanning the Electra fork transition",
```

Comment:
I guess this should be `Fulu` not `Electra`

Reply:
I think this needs to be rephrased. I believe it's trying to say that the request spans over Electra & Fulu slots.